packages/grid/_modules_/grid/models/gridOptions.tsx

```diff
@@ -346,7 +354,8 @@ export const DEFAULT_GRID_OPTIONS: GridOptions = {
   headerHeight: 56,
   columnBuffer: 2,
   rowsPerPageOptions: [25, 50, 100],
-  pageSize: 100,
+  pageSize: 80,
```

Comment:
Why change this default value?

Reply:
My bad I meant to change the newly added `scrollBottomThreshold` prop. Reverted